packages/modules/src/modules/epics/ContributionsEpicButtons.tsx

```diff
@@ -173,7 +233,7 @@ export const ContributionsEpicButtons = ({
         (secondaryCta?.type === SecondaryCtaType.Custom && isSupportUrl(secondaryCta.cta.baseUrl));
 
     return (
-        <div ref={setNode} css={buttonWrapperStyles} data-testid="epic=buttons">
+        <div ref={setNode} css={buttonWrapperStyles(showApplePayButton)} data-testid="epic=buttons">
             {!isReminderActive && (
```

Comment:
I'd quite like to look at the code below together, I think the approach of pass `showApplePayButton` to create variations of all the existing components (`PrimaryCtaButton`, `SecondaryCtaButton` etc.) has meant you've had to refactor these existing components, I do wonder if this is avoidable and whether we can leave these components as they were before, whilst simulatneously introducing the `ButtonApplePay` component.

Reply:
Rebuilt/simplified with 2 further components `PrimaryCtaButtonApplePay` and `SecondaryCtaButtonApplePay` as we discussed. Styling also used to display the third Reminder Cta below desktop when ApplePay displayed.